.azure-pipelines/playbooksValidations.yaml

```diff
@@ -0,0 +1,12 @@
+jobs:
```

Comment:
How did you do it not required?
![image](https://user-images.githubusercontent.com/38046493/119561012-5223c580-bdad-11eb-94ab-5d802474e8ef.png)

Reply:
I have no idea :(

Reply:
I'm fine with this not being required check, at least at the start, so we'll be able to verify these new validations work fine and don't block contribution. But I have no idea what controls it so I don't know how to change it to required later on